modules/examiner/jsx/examiner.js

```diff
@@ -0,0 +1,349 @@
+      }
+    }
+    formObject.append('fire_away', 'Add');
+    $.ajax({
+      type: 'POST',
+      url: loris.BaseURL + '/examiner/',
+      data: formObject,
+      cache: false,
+      contentType: false,
+      processData: false,
+      success: data => {
+        swal('Success!', 'Examiner added.', 'success');
+        this.fetchData();
+      },
+      error: error => {
+        console.error(error);
+        let message = error.responseText;
+        swal('Error!', message, 'error');
+      }
+    });
+  }
+
+  /**
+   * Modify behaviour of specified column cells in the Static Data Table component
+   *
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {array} rowData - array of cell contents for a specific row
+   * @param {array} rowHeaders - array of table headers (column names)
+   *
+   * @return {*} a formated table cell for a given column
+   */
+  formatColumn(column, cell, rowData, rowHeaders) {
+    if (this.state.data.hiddenHeaders.indexOf(column) > -1) {
+      return null;
+    }
+    // Create the mapping between rowHeaders and rowData in a row object.
+    let row = {};
+    rowHeaders.forEach((header, index) => {
+      row[header] = rowData[index];
+    });
+
+    switch (column) {
+      case 'Examiner': {
+        let url = loris.BaseURL + '/examiner/editExaminer/?identifier=' +
+                  row.ID;
+        return (
+          <td>
+            <a href ={url}>{cell}</a>
+          </td>
+        );
+      }
+      case 'Radiologist': {
+        let radiologist = 'No';
+        if (row.Radiologist === '1') {
+          radiologist = 'Yes';
+        }
+        return (
+          <td>
+            {radiologist}
+          </td>
+        );
+      }
+      case 'Certification':
+        if (row.Certification === null) {
+          return (
+            <td>
+              None
+            </td>
+          );
+        }
+        return (
+          <td>
+            {cell}
+          </td>
+        );
+      default:
+        return (
+          <td>
+            {cell}
+          </td>
+        );
+    }
+  }
+
+  render() {
+    // Waiting for async data to load
+    if (!this.state.isLoaded) {
+      return (
+        <Loader/>
+      );
+    }
+
+    if (this.state.data.form) {
+      // Make elements required
+      this.state.data.form.addName.required = true;
+      this.state.data.form.addSite.required = true;
+
+      // Set value in Add Examiner form
+      this.state.data.form.addName.value = this.state.formData.addName;
+      this.state.data.form.addRadiologist.value =
+          this.state.formData.addRadiologist;
+      this.state.data.form.addSite.value = this.state.formData.addSite;
+
+      // Set onUserInput in Add Examiner form
+      this.state.data.form.addName.onUserInput = this.setFormData;
+      this.state.data.form.addRadiologist.onUserInput = this.setFormData;
+      this.state.data.form.addSite.onUserInput = this.setFormData;
```

Comment:
After testing this PR, I _highly_ recommended defining the add examiner form fields in the the jsx files, rather than sending them from the server. This will eliminate the need for this entire section from `231` to `245`. 

If those changes are out of the scope of this PR, then this section should be redistributed into the `fetchData()` and `setFormData` functions as follows:

```  
  /**                                                                           
   * Retrieve data from the provided URL and save it in state                   
   */                                                                           
  fetchData() {                                                                 
    const self = this;                                                          
    $.ajax(this.props.DataURL, {                                                
      method: 'GET',                                                            
      dataType: 'json',                                                         
      success: data => {                                                        
        loris.hiddenHeaders = data.hiddenHeaders ? data.hiddenHeaders : [];        
        data.form.addName.required = true;                                      
        data.form.addSite.required = true;                                                                                                         
        data.form.addName.onUserInput = this.setFormData;                       
        data.form.addRadiologist.onUserInput = this.setFormData;                
        data.form.addSite.onUserInput = this.setFormData;                       
                                                                                
        self.setState({                                                         
          data: data,                                                           
          isLoaded: true                                                        
        });                                                                     
      },                                                                        
      error: error => {                                                         
        console.error(error);                                                   
      }                                                                         
    });                                                                         
  }   
```

```  
  /**                                                                           
   * Store the value of the element in this.state.formData                      
   *                                                                            
   * @param {string} formElement - name of the form element                     
   * @param {string} value - value of the form element                          
   */                                                                           
  setFormData(formElement, value) {                                             
    let formData = this.state.formData;                                         
    let data     = this.state.data;                                             
    
    formData[formElement] = value;                                                   
    data.form.addName.value = formData.addName;                                 
    data.form.addRadiologist.value = formData.addRadiologist;                   
    data.form.addSite.value = formData.addSite;                                 
    this.setState({data, formData});                                            
  } 
```

Reply:
@HenriRabalais @driusan that makes sense, and is definitely within the scope of this PR. I had initially defined the fields in jsx as recommended above, but decided to use what already exists in the php class, as is done for the menu filters. was just following existing practices but you're right, the jsx way is definitely the better way